cloud/docker/docker_compose

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+
+# (c) 2015, Sam Yaple <sam@yaple.net>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
```

Comment:
Aren't required: false and default=None the defaults here? Or do they need to be explicitly set?

I see no best practice for this and many modules have missing "required" and "default" fields if it is a requirement.